gtfs-realtime/spec/en/reference.md

```diff
@@ -554,13 +561,132 @@ A localized image URL mapped to a language.
+| **level_id** | [string](https://protobuf.dev/programming-guides/proto2/#scalar) | Optional | One |  See definition of [stops.level_id](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stopstxt) in (CSV) GTFS. |
+| **platform_code** | [TranslatedString](#message-translatedstring) | Optional | One |  See definition of [stops.platform_code](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stopstxt) in (CSV) GTFS. |
+
+## _enum_ WheelchairBoarding
+
+**Values**
+
+| _**Value**_ | _**Comment**_ |
+|-------------|---------------|
+| **UNKNOWN** | No accessibility information for the stop. |
+| **AVAILABLE** |  Some vehicles at this stop can be boarded by a rider in a wheelchair. |
+| **NOT_AVAILABLE** | Wheelchair boarding is not possible at this stop. |
+
+## _message_ TripModifications
+
+A `TripModifications` message identifies a list of similar trips which are all affected by particular modifications, such as a detour.
+
+<br><br>**Caution:** this field is still **experimental**, and subject to change. It may be formally adopted in the future.
+ 
+[More about Trip Modifications...](trip-modifications.md)
+
+**Fields**
+
+| _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_ |
+|------------------|------------|----------------|-------------------|-------------------|
+| **selected_trips** | [SelectedTrips](#message-selectedtrips) | Required | Many | A list of selected trips affected by this TripModifications. Needs to contain at least one `SelectedTrips`. If the value `start_times` is set, a maximum of one `SelectedTrips` with one trip_id can be listed.  |
+| **start_times** | [string](https://protobuf.dev/programming-guides/proto2/#scalar) | Optional | Many | A list of start times in the real-time trip descriptor for the trip_id defined in trip_ids. Useful to target multiple departures of a trip_id in a frequency-based trip. |
+| **service_dates** | [string](https://protobuf.dev/programming-guides/proto2/#scalar) | Required | Many | Dates on which the modification occurs, in the YYYYMMDD format. A trip_id will only be modified if it runs on a given service date; the trip IS NOT required to run on all of the service dates. Producers SHOULD only transmit detours occurring within the next week. The dates provided should not be used as user-facing information, if a user-facing start and end date need to be provided, they can be provided in the linked service alert with `service_alert_id` |
+| **modifications** | [Modification](#message-modification) | Required | Many | A list of modifications to apply to the affected trips. |
+
+## _message_ Modification
+
+A `Modification` message replaces a span of n stop times from each affected trip starting at `start_stop_selector`.
```

Comment:
It's possible for no stop times to be replaced.

```suggestion
A `Modification` message describes changes to each affected trip starting at `start_stop_selector`.
```

Reply:
In that case we meant that n could be 0, but your language is clearer.